client/components/common/tce-core/ElementList.vue

```diff
@@ -52,11 +61,11 @@ export default {
   },
   computed: {
     options() {
-      return {
+      return Object.assign(this.dragOptions, {
```

Comment:
```suggestion
      options: vm => Object.assign(vm.dragOptions, {
      handle: '.drag-handle',
      scrollSpeed: 15,
      scrollSensitivity: 125
    }),
```

Also, you can write component `data` ☝️  as a one-liner to be consistent with the rest of the application :bowtie: